rebar.config

```diff
@@ -137,6 +137,10 @@
                    {copy, "data/aecore/.fortuna/accounts.json", "data/aecore/.fortuna/accounts.json"},
                    {copy, "data/aecore/.fortuna/accounts_uat.json", "data/aecore/.fortuna/accounts_uat.json"},
                    {copy, "data/aecore/.fortuna/accounts_test.json", "data/aecore/.fortuna/accounts_test.json"},
+                   {mkdir, "data/aecore/.lima"},
```

Comment:
I wonder if these packaging lines can be split in separate line.

Reply:
Actually, they are needed for the tests to pass. So, no.